test/v1beta1/note_test.go

```diff
@@ -257,7 +257,13 @@ func TestNote(t *testing.T) {
 
 				Expect(err).ToNot(HaveOccurred())
 				Expect(res.Notes).To(HaveLen(pageSize))
-				Expect(res.NextPageToken).ToNot(BeEmpty())
+
+				isLastPage := len(res.Notes)+len(foundNotes) == len(batch.Notes)
```

Comment:
If the `pageSize` on line 251 is chosen as 2 for every run, we'll hit the last page of notes and the page token will be empty.